docs/source/developer_guide.rst

```diff
@@ -712,15 +712,15 @@ dictionary with the free\_RVs being sampled now has a new value (if
 accepted, see
 `here <https://github.com/pymc-devs/pymc3/blob/6d07591962a6c135640a3c31903eba66b34e71d8/pymc3/step_methods/compound.py#L27>`__
 and
-`here <https://github.com/pymc-devs/pymc3/blob/master/pymc3/step_methods/compound.py#L41>`__).
+`here <https://github.com/pymc-devs/pymc3/blob/main/pymc3/step_methods/compound.py#L41>`__).
```

Comment:
I'm not sure this is pointing to the right line, and it will have to be updated from time to time. I think ideally this would point to the documentation and the documentation would have the source code (so you can link to methods independently of the line they are in, for example https://arviz-devs.github.io/arviz/_modules/arviz/stats/stats.html#hdi.

Reply:
The entire file has just 72 lines and the compound step docs are linked a few lines blow.
I guess we can just remove the line number.
```suggestion
`here <https://github.com/pymc-devs/pymc3/blob/main/pymc3/step_methods/compound.py>`__).
```